### ISSUE TITLE
Stuck at "Please wait..."

### ISSUE THREAD

**Issue by keverets** [davidar/tensor#11]
After initial installation, running the application on Android (Tensor 0.3 from f-droid) brings me to a sign-in page. I sign in with the username & password I recall using on matrix.org in the past, but then the application changes to a "Please wait..." view with a rotating symbol on it, but no information about what I'm waiting for (network? authentication? something else?).

It has been in this state for approximately 5 minutes.

Perhaps giving the user some feedback to the user as to what's happening in the sign-in flow (especially if there's a large delay) would be helpful.

**Comment by davidar** [comment]
Yeah, that usually happens when signing in fails for some reason, agreed that there should be better feedback.

**Comment by WGH-** [comment]
I've also encountered this.

Moreover, I have absolutely no idea what went wrong, since, according to strace output, tensor simply did nothing after receiving DNS reply.
